queries.md

```diff
@@ -4,40 +4,146 @@
 
 ### 1. All the companies that it's name match 'Babelgum'. Retrieve only their `name` field.
 
+filter: { name: {$eq: "Babelgum"}}
+project: {name: 1}
+
 ### 2. All the companies that have more than 5000 employees. Limit the search to 20 companies and sort them by **number of employees**.
 
-### 3. All the companies founded between 2000 and 2005, both years included. Retrieve only the `name` and `founded_year` fileds.
+filter: { number_of_employees: {$gte: 5000}}
+sort: {number_of_employees : -1}
+limit: 20
 
+### 3. All the companies founded between 2000 and 2005, both years included. Retrieve only the `name` and `founded_year` fileds.
+filter: { founded_year: {$gte: 2000, $lte: 2005}}  
+project: {name: 1, founded_year: 1, _id: 0}
 ### 4. All the companies that had an IPO of more than 100.000.000 and have been founded before 2010. Retrieve only the `name` and `ipo` fields.
+filter: {$and: [{'ipo.valuation_amount': {$gt: 100000000}}, {founded_year: {$lte: 2010}}]}
+project: {name: 1}, {ipo: 1}
```

Comment:
For the 'project', you can simply put both arguments inside the same area, separated by a coma like so:
`{name: 1, ipo: 1}`
And don't forget to tell it not to display the _id field, or it will do it by default!
`{name: 1, ipo: 1, _id: 0}`